app/app.go

```diff
@@ -253,6 +255,10 @@ func (a *app) DeleteIOTHubDevice(ctx context.Context, deviceID string) error {
 			}
 			err = a.hub.DeleteDevice(ctx, cs, deviceID)
 			if err != nil {
+				if htErr, ok := err.(client.HTTPError); ok &&
+					htErr.Code() == http.StatusNotFound {
+					continue
```

Comment:
Not super important, but this is not covered by unit tests.

Reply:
I'll follow up in my upcoming PR with acceptance tests.